device/pcie/pci_device.cpp

```diff
@@ -45,6 +46,22 @@ const uint32_t HUGEPAGE_REGION_SIZE = 1 << 30; // 1GB
 
 using namespace tt;
 using namespace tt::umd;
+using tt::umd::semver_t;
+
+static semver_t read_kmd_version() {
```

Comment:
Can you make this a private function of PCIDevice instead?

Reply:
yes, fixed